### PR TITLE
Reduce number of scarb build calls in tests

### PR DESCRIPTION
**Pull request by MrDenkoV**
Fix (#608)

### CODE REVIEW

**Comment by mkaput** [comment]
What is missing still here, that this is a draft?

**Comment by MrDenkoV** [comment]
I was thinking #137 will be added soon and wanted to wait for it to be done to change a couple more tests. I will change the number in Todo to #137 and make the PR ready.
@mkaput

**Comment by mkaput** [comment]
> I was thinking #137 will be added soon

Not gonna happen

File: scarb/tests/build.rs [hunk 1]
```diff
@@ -69,7 +69,7 @@ fn compile_with_syntax_error() {
         .build(&t);
 
     Scarb::quick_snapbox()
-        .arg("build")
+        .arg("build") // Todo(#137) check
```
Comment (mkaput) [inline scarb/tests/build.rs:72]: Could you please use the standard pattern for todo comments that we use elsewhere? :)

**Review by mkaput — CHANGES_REQUESTED** [review]

**Review by mkaput — APPROVED** [review]

**Pull request merged**